scripts/bash/getInternalDependencies.sh

```
@@ -37,10 +60,10 @@ internal_dependencies=("${array_without_gaps[@]}")
 unset array_without_gaps
 
 # recursively build up an array of all internal dependencies this package depends on
-for dependency in ${internal_dependencies[@]}; do
+for dependency in "${internal_dependencies[@]}"; do
 
   nested_dependencies=($(${DIR}/getInternalDependencies.sh "${package_path}/../${dependency}" ${dependencies_already_visited[@]} ${internal_dependencies[@]} ))
```

Shellcheck (0.10.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)